test/unit/specs/hive/basic-store.js

```diff
@@ -0,0 +1,2237 @@
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+
+import Backendless, { APP_PATH, forTest, prepareMockRequest } from '../../helpers/sandbox'
+import { HiveTypes } from '../../../../src/hive/constants'
+import { HiveStore } from '../../../../src/hive/stores/base-store'
+
+describe('Hive Store Class', function() {
+  forTest(this)
+
+  const hiveName = 'test'
+  const storeKey = 'testStoreKey'
+
+  const getStoreUrl = storeName => `${APP_PATH}/hive/${hiveName}/${storeName}`
```

Comment:
would be better to compose the string in each test

Reply:
done